src/components/Date/Date.unit.spec.jsx

```diff
@@ -43,16 +18,13 @@ describe('<Date>', () => {
     date.month.dirty = true;
     date.day.dirty = true;
 
-    const tree = shallow(
+    const { getByText } = render(
       <Date required date={date} onValueChange={() => {}} />,
     );
 
-    expect(tree.find('.usa-input-error').exists()).to.be.true;
-    expect(tree.find('.usa-input-error-message').text()).to.equal(
-      'Error Please provide a response',
-    );
-    tree.unmount();
+    getByText(/Please provide a response/);
```

Comment:
I think that the `getBy*` functions throw an error if the thing isn't found, but do you think it makes sense to also have an `expect` for these tests?

Reply:
I was going back and forth. Having an `expect` in there is probably clearer, even if it's not _cleaner._ :man_shrugging: